engine/src/time.cpp

```diff
@@ -9,26 +9,28 @@ namespace gen
 		namespace
 		{
 			Clock::time_point g_start{Clock::now()};
-			double g_deltaTime;
-			double g_timeScale;
+			float g_deltaTime;
```

Comment:
Why are these stored as floats instead of chrono durations?

Reply:
This code is from @Kouros26 I was just updating the code to use floats instead of doubles without restructuring the core of it too much to make the compiler happy. I'll likely make changes to this code in another PR.